qiskit/providers/ibmq/api_v2/rest/backend.py

```diff
@@ -36,10 +37,19 @@ def __init__(self, session, backend_name):
         self.backend_name = backend_name
         super().__init__(session, '/devices/{}'.format(backend_name))
 
-    def properties(self):
+    def properties(self, filter=None):
```

Comment:
You should update the docstring to add the new parameter (this applies to some other functions you modified as well).